x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/api.ts

```diff
@@ -84,14 +85,34 @@ export const fetchQueryRuleRegistryAlerts = async <Hit, Aggregations>({
  *
  * @throws An error if response is not OK
  */
-export const updateAlertStatus = async ({
+export const updateAlertStatusByQuery = async ({
   query,
   status,
   signal,
-}: UpdateAlertStatusProps): Promise<estypes.UpdateByQueryResponse> =>
+}: UpdateAlertStatusByQueryProps): Promise<estypes.UpdateByQueryResponse> =>
   KibanaServices.get().http.fetch(DETECTION_ENGINE_SIGNALS_STATUS_URL, {
     method: 'POST',
-    body: JSON.stringify({ conflicts: 'proceed', status, ...query }),
+    body: JSON.stringify({ conflicts: 'proceed', status, query }),
+    signal,
+  });
+
+/**
+ * Update alert status by signalIds
+ *
+ * @param signalIds List of signal ids to update
+ * @param status to update to('open' / 'closed' / 'acknowledged')
+ * @param signal AbortSignal for cancelling request
+ *
+ * @throws An error if response is not OK
+ */
+export const updateAlertStatusByIds = async ({
+  signalIds,
+  status,
+  signal,
+}: UpdateAlertStatusByIdsProps): Promise<estypes.BulkResponse> =>
+  KibanaServices.get().http.fetch(DETECTION_ENGINE_SIGNALS_STATUS_URL, {
+    method: 'POST',
+    body: JSON.stringify({ status, signal_ids: signalIds }),
```

Comment:
Do we need ```conflicts: 'proceed'``` here as well?

Reply:
`conflicts ` property isn't supported by the `_bulk` API.